examples/module-federation/app1/webpack.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   entry: "./src/index",
   mode: "development",
   devServer: {
-    contentBase: path.join(__dirname, "dist"),
+    static: path.join(__dirname, "dist"),
```

Comment:
Need to update the webpack-dev-server version to v4 as well in package.json here.

Reply:
https://github.com/webpack/webpack.js.org/blob/98973c7bb1fb6ef80b6428e01a9bf597412b075c/examples/module-federation/app1/package.json#L22

https://github.com/webpack/webpack.js.org/blob/98973c7bb1fb6ef80b6428e01a9bf597412b075c/examples/module-federation/app2/package.json#L21

Reply:
Once updated, you can check this link https://stackblitz.com/github/webpack/webpack.js.org/tree/8e4e3207e24932c3f64b44ce0ac8a5e8092605ad/examples/module-federation to see if the example works.

Reply:
lgtm 
![image](https://user-images.githubusercontent.com/3165436/130277418-4870a301-d753-4b40-96b4-f63f2b28d173.png)

Reply:
Awesome.